lib/jellyfish/room_service.ex

```diff
@@ -80,9 +102,21 @@ defmodule Jellyfish.RoomService do
     GenServer.call(__MODULE__, {:delete_room, room_id})
   end
 
-  @spec get_resource_usage() :: {Node.t(), integer()}
+  @spec get_resource_usage() :: %{
+          node: Node.t(),
+          forwarded_tracks_number: integer(),
+          rooms_number: integer()
+        }
   def get_resource_usage() do
-    list_rooms() |> Enum.count() |> then(&{Node.self(), &1})
+    room_ids = get_rooms_ids()
+
+    Enum.each(room_ids, &Room.request_forwarded_tracks_number_in_room(&1))
```

Comment:
I get the idea but I think it will be easier to make `request_forwarded_tracks_number_in_room` synchronous and spawn multiple tasks to get it in a concurrent way. You can use Task.away_many